syntax = "proto3";

package proto_renderer;

message Map {
    bytes     description = 1;
}